modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,558 @@
+= Production Readiness Checklist 
+
+Before running production workloads on Redpanda, work through the readiness checklists provided here to ensure that your cluster is deployed and configured according to Redpanda Data recommendations. The checklists help ensure that you are set up for success. There are three levels of production readiness:
+- Level 1 
+** Description of the level.
+- Level 2
+** Description of the level.
+- Level 3 
+** Description of the level.
+
+Satisfying the criteria presented in these checklists helps ensure that you are ready to run mission critical workloads in the largest of enterprises.
+
+Redpanda provides multiple resources for deploying a production cluster, and recommends that you utilize the xref:deploy:deployment-option/self-hosted/manual/production/production-deployment-automation.adoc[automated deployment instructions] in the Redpanda Ansible collection. If you cannot deploy with Ansible, use the xref:deploy:deployment-option/self-hosted/manual/production/production-deployment.adoc[manual deployment instructions].
+
+== Checklists
+
+=== Level 1 checklist
+
+[%interactive]
+* [ ] <<redpanda-license>>
+* [ ] <<cluster-health>>
+* [ ] <<production-mode>>
+* [ ] <<redpanda-lmr>>
+* [ ] <<cpu-memory>>
+* [ ] <<mounted-disks>>
+* [ ] <<auth-enable>>
+* [ ] <<super-users>>
+* [ ] <<tls-enabled>>
+* [ ] <<redpanda-tuners>>
+* [ ] <<disk-perf>>
+* [ ] <<hostnames-interfaces>>
+* [ ] <<continuous-db>>
+* [ ] <<debug-bundle>>
+* [ ] <<topic-rf>>
+* [ ] <<maintenance-mode>>
+* [ ] <<decom-state>>
+
+
+
+=== Level 2 checklist
+
+[%interactive]
+* [ ] Dev / test environment available?
+* [ ] Metrics being captured and stored
+* [ ] System logs being captured and stored
+* [ ] Upgrade policy defined and implemented
+* [ ] Rack awareness configured correctly
+
+
+=== Level 3 checklist
+
```

Comment:
For Level 3 readiness, confirm that:

* You are proactively monitoring mission-critical workloads, business continuity solutions, and integration into enterprise security systems
* Your enterprise is ready to run mission-critical workloads
* 
The following steps can help you achieve Level 3 readiness: